src/Tool/ConstructorParameterResolverInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\ServiceManager\Tool;
+
+use Psr\Container\ContainerInterface;
+
+interface ConstructorParameterResolverInterface
```

Comment:
Does this mean that more resolvers will be added in the future? (I am thinking here of forms, for example.)

Reply:
Actually not, no. This is just meant to be used with services, IMHO more complex services with scalar argunemt constructors or with a "injection strategy" should stay as is. 

At least I am not planning of supporting more than services available via the servicemanager. 
So anything pluginmanager related wont work with reflection as of now as well. 
But at least having that interface will open up 3rd party to implement more complex resolving strategies.

Reply:
> But at least having that interface will open up 3rd party to implement more complex resolving strategies.

👍🏻